src/api/app/views/webui2/webui/patchinfo/_side_elements.html.haml

```diff
@@ -0,0 +1,32 @@
+%ul.list-unstyled
+  - if patchinfo.block
+    %li
+      %div{ data: { toggle: 'collapse', target: '#blocked-reason-details' },
+            aria: { controls: 'blocked-reason-details', expanded: 'false' }, role: 'button' }
+        %i.fas.fa-info-circle.text-info
+        %span Blocked Reason
+        %i.fas.fa-chevron-left.expander{ title: 'Show blocked reason' }
+        %i.fas.fa-chevron-down.collapser{ title: 'Hide blocked reason' }
+      .text-muted.collapse#blocked-reason-details
+        = patchinfo.block_reason.presence || 'No reason entered.'
+  - if packager
+    %li
+      %i.fas.fa-info-circle.text-info
+      %span Submitted from
+      %span= user_with_realname_and_icon(packager)
+  - if patchinfo.version
+    %li
+      %i.fas.fa-info-circle.text-info
+      %span Version #{patchinfo.version}
+  - if patchinfo.relogin_needed
+    %li
+      %i.fas.fa-info-circle.text-warning
```

Comment:
When using `text-warning`, we should use the icon `fa-exclamation-triangle` to be consistent with the other views. See the [patterns](https://obs-patterns.netlify.com/atoms/icons.html) for details.